R/help_create_blogpost.R

```diff
@@ -0,0 +1,151 @@
+library(stringr)
+library(tidyverse)
```

Comment:
can we just load stringer, dplyr and any others?  tidyverse has 22 packages in it I think and we only use a few